src/redisjson.rs

```diff
@@ -52,16 +52,35 @@ impl RedisJSON {
     pub fn to_string(&self, path: &str) -> Result<String, Error> {
         eprintln!("Serializing back to JSON");
 
+        let s = match self.get_doc(path)? {
+            Some(doc) => serde_json::to_string(&doc)?,
+            None => String::new()
+        };
+
+        Ok(s)
+    }
+
+    pub fn str_len(&self, path: &str) -> Result<usize, Error> {
+        let s = match self.get_doc(path)? {
+            Some(doc) => {
+                if doc.is_string() {
+                    serde_json::to_string(&doc)?.len() - 2 // removes ""
+                } else {
+                    0 // the value is not a String
```

Comment:
Wouldn't the Redis-like behavior be to return a `WRONGTYPE` error, similar to calling `STRLEN` on a non-string key?